src/components/themed/TransactionListTop.js

```diff
@@ -337,7 +336,7 @@ export const TransactionListTop = connect(
     // Crypto Amount Formatting
     const currencyDenomination = getDenomination(selectedCurrencyCode, state.ui.settings)
     const cryptoAmount: string = convertNativeToDenomination(currencyDenomination.multiplier)(balance) // convert to correct denomination
-    const cryptoAmountFormat = cryptoAmount && !bns.eq(cryptoAmount, '0') ? intl.formatNumber(cryptoAmount.replace(/0+$/, '')) : '0' // only cut off trailing zeroes (to the right of significant figures)
+    const cryptoAmountFormat = intl.formatNumber(truncateDecimalTrailingZeros(cryptoAmount))
```

Comment:
No need to create a new routine. biggystring already truncates decimals on all operations. Just do the following:

`const cryptoAmountFormat = intl.formatNumber(bns.add(cryptoAmount, '0'))`

Reply:
fixed